packages/terra-graphs/src/components/Combination/CombinationGraph.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+import '../Line/LineGraph.module.scss';
+import '../Bar/BarGraph.module.scss';
+import '../PairedResult/PairedResultGraph.module.scss';
+
+const propTypes = {
+  /**
+   * ID for graph created.
+   */
+  graphID: PropTypes.string.isRequired,
+  /**
+   * Graph configuration object
+   */
+  graphConfig: PropTypes.object.isRequired,
+  /**
+   * data to be displayed in graph
+   */
+  dataset: PropTypes.arrayOf(PropTypes.object),
+  /**
+   * timeout to display multiple data contents in specific time interval.
+   */
+  timeout: PropTypes.arrayOf(PropTypes.number),
+};
+
+// eslint-disable-next-line consistent-return
```

Comment:
I think I agree with the linter on this one. I'd refactor this to return the Carbon-generated output but not load the config, that way you can get some consistent functional behavior here.

```jsx
const datasetLoader = (dataset) => {
  let graphType = dataset.graphType.toUpperCase();
  graphType = graphType.replaceAll(' ', '');
  
  let apiData;
  switch (graphType) {
    case 'LINE':
      apiData = Carbon.api.line(dataset);
      break;
    ...
    default:
      apiData = undefined;
    }
    
    return apiData;
};
```

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/150/commits/fcf005dd85028592d80e122d575530689fb294ac#diff-69b4098fd145f5ca79edeeb7a5fbeeec17981c0e5804acc94bca187ca411f4e0R28).